setup.cfg

```diff
@@ -0,0 +1,31 @@
+[pycodestyle]
```

Comment:
Do we use `pycodestyle`? I don't see it is run automatically and I don't see it in the requirements 🤔